app/src/main/res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <string name="pokemon029">Nidoran♀</string>
-    <string name="pokemon032">Nidoran♂</string>
+    <string name="pokemon029" translatable="false">Nidoran♀</string>
+    <string name="pokemon032" translatable="false">Nidoran♂</string>
```

Comment:
It's being used in initOcr, but since we're dropping Jap support, I'm moving this to code.